### PR TITLE
feature/school details

### PR DESCRIPTION
**Pull request by NikGorbunov**

### CODE REVIEW

File: src/components/ThemeRegistry/theme.js [hunk 1]
```diff
@@ -61,12 +61,12 @@ const theme = createTheme({
       fontWeight: 700,
       fontFamily: montserrat.style.fontFamily,
     },
-    subheading: ({ theme }) => ({
+    subtitle1: {
```
Comment (shuryhin-oleksandr) [inline src/components/ThemeRegistry/theme.js:64]: Add custom typography variant

File: src/app/(homepage)/HomepageTabs.js [hunk 2]
```diff
@@ -12,30 +12,30 @@ import { useTheme } from "@mui/material/styles";
...
-      <Typography mt={2} variant="body1" color="text.secondary">
-        You shouldn’t have your work cut out finding and syncing holidays,
-        birthdays and contacts. Once you&apos;re on board, you&apos;ll have access to
-        crucial information like school holiday schedules, classmates&apos;
-        birthdays, and parents&apos; contact details at your fingertips.
-      </Typography>
-    </Box>
-  );
+
+const SchoolDetail = {
```
Comment (shuryhin-oleksandr) [inline src/app/(homepage)/HomepageTabs.js:16]: Important convention in React is that CapitalCase are only components. This should be UPPER_CASE as a constant data.

File: src/app/(homepage)/HomepageTabs.js [hunk 3]
```diff
@@ -53,6 +53,73 @@ const CustomTab = styled(Tab)(({ theme }) => ({
...
+        </Typography>
+        <Typography mb={3} variant="h3">
+          {data.header}
+        </Typography>
+        <Typography mb={6.7} variant="body1" color="text.secondary">
+          {data.description}
+        </Typography>
+        <Button variant="outlined" color="orange" size="large" sx={{
+          textTransform: 'none',
+          fontSize: theme.typography.htmlFontSize,
```
Comment (shuryhin-oleksandr) [inline src/app/(homepage)/HomepageTabs.js:105]: Same here

File: src/app/(homepage)/HomepageTabs.js [hunk 4]
```diff
@@ -53,6 +53,73 @@ const CustomTab = styled(Tab)(({ theme }) => ({
...
+        }}>
+          {data.button}
+        </Button>
+      </Box>
+      <Box
+        component="img"
```
Comment (shuryhin-oleksandr) [inline src/app/(homepage)/HomepageTabs.js:111]: It's best to use Next Image component for optimized image rendering

File: src/components/ThemeRegistry/theme.js [hunk 5]
```diff
@@ -61,12 +61,12 @@ const theme = createTheme({
...
       fontFamily: montserrat.style.fontFamily,
     },
-    subheading: ({ theme }) => ({
+    subheading: {
       fontFamily: montserrat.style.fontFamily,
       fontSize: "1rem", // 16px
       fontWeight: 700,
-      color: theme.palette.orange.main,
-    }),
+      color: colors.orange.main,
```
Comment (shuryhin-oleksandr) [inline src/components/ThemeRegistry/theme.js:68]: Use palette better to keep it decoupled from colors as possible.

File: src/components/ThemeRegistry/colors.js [hunk 6]
```diff
@@ -100,6 +100,6 @@ const colorsDefault = Object.fromEntries(
 
 export const colors = {
   ...colorsDefault,
-  grey: { ...grey, light: grey[500], main: grey[600], dark: "#000000", contrastText: "#FFFFFF" },
+  grey: { ...grey, light: grey[400], main: grey[600], dark: "#000000", contrastText: "#FFFFFF" },
```
Comment (shuryhin-oleksandr) [inline src/components/ThemeRegistry/colors.js:103]: Hm, I would keep it as it was, as this setting has side effects, grey button hover color, for example.

File: src/components/ThemeRegistry/theme.js (hunk 5 above)
Comment (oleksandr-shvab) [reply src/components/ThemeRegistry/theme.js:68]: Looks like only this solution work

File: src/app/(homepage)/HomepageTabs.js [hunk 7]
```diff
@@ -53,6 +54,75 @@ const CustomTab = styled(Tab)(({ theme }) => ({
...
+          {data.header}
+        </Typography>
+        <Typography mb={6.7} variant="body1" color="text.secondary">
+          {data.description}
+        </Typography>
+        <Button variant="outlined" color="orange" size="large" >
+          {data.button}
+        </Button>
+      </Box>
+      <Box
```
Comment (shuryhin-oleksandr) [inline src/app/(homepage)/HomepageTabs.js:107]: Why wrap image in Box, do not set these directly to image?

File: src/app/(homepage)/HomepageTabs.js [hunk 8]
```diff
@@ -53,6 +54,75 @@ const CustomTab = styled(Tab)(({ theme }) => ({
...
+        birthdays and contacts. Once you&apos;re on board, you&apos;ll have access to
+        crucial information like school holiday schedules, classmates&apos;
+        birthdays, and parents&apos; contact details at your fingertips.
+      </Typography>
+    </Box>
+  );
+}
+
+
+function ContentPanel({ type }) {
```
Comment (shuryhin-oleksandr) [inline src/app/(homepage)/HomepageTabs.js:78]: image
Spacing between sections sucks (

**Pull request merged**